src/processing/script/transformers/property-get.ts

```diff
@@ -32,6 +32,11 @@ const transformer: Transformer<MemberExpression> = {
         if (node.property.type === Syntax.Identifier && !shouldInstrumentProperty(node.property.name))
             return false;
 
+        // Skip: object.#prop
+        // @ts-ignore
+        if (node.property.type === Syntax.PrivateIdentifier)
```

Comment:
Have you tried to update  `estree`? Perhaps there won't be any problems.

Reply:
Yes, I have tried, and this leads to numerous conflicts in TypeScript

Reply:
Okay, let's remove comment `// Skip: object.#prop` or refactor it in a way: `NOTE: <describing case>`